src/Eurofurence.App.Server.Services/PushNotifications/FirebaseChannelManager.cs

```diff
@@ -20,6 +21,7 @@ public class FirebaseChannelManager : IFirebaseChannelManager
     {
         private readonly IDeviceIdentityService _deviceService;
         private readonly IRegistrationIdentityService _registrationService;
+        private readonly AppDbContext _db;
```

Comment:
Please use `_appDbContext` instead of `_db` as that's how it's referred to in all other services. (=